bundles/org.openhab.binding.netatmo/src/main/resources/ESH-INF/config/config.xml

```diff
@@ -142,7 +148,7 @@
 		</parameter>
 	</config-description>
 
-	<config-description uri="thing-type:netatmo:nawelcomecamera">
+	<config-description uri="thing-type:netatmo:camera">
```

Comment:
Does this change effect existing users?

Reply:
That is a good question.
How can this change affect existing users ? You think the config url is stored in the JSON database ?

Reply:
IIRC it will not. We are persisting the Thing configuration. Not the config descriptions.

Reply:
Hm, I'm not sure. Are you sure enough?

When not I think we have to revert that (but it is a result of a code review) or we have to test/confirm that (but I'm missing a Welcome camera which would necessary to test that).